api-flask/docker-compose.test.yml

```diff
@@ -15,10 +15,19 @@ services:
     # Infinite loop, to keep it alive, for debugging
     # command: bash -c "while true; do echo 'sleeping...' && sleep 10; done"
     #
+    depends_on:
+      - testdb
+    networks:
```

Comment:
why did you add this network here? by default containers in the same file will share the same network

Reply:
but it might be interesting to do that indeed, if we also add the same network-name to the database initiated in the alerting module to make sure that we can connect to it